src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -52,5 +52,7 @@ internal class LocalizableStrings
                                         Logs are written to the provided file.";
 
         public const string CmdNoBuildDescription = @"Do not build project before testing.";
+
+        public const string RunSettingsArgsHelpText = @"Any extra options that should be passed to MSBuild or adapter. See 'dotnet msbuild -h' and 'dotnet vstest --help' for available options.";
```

Comment:
@blackdwarf Please suggest the documentation text for dotnet test. 

Here is how a dotnet command would look like :
dotnet test foo.csproj -- logger trx {any argument to pass to msbuild} -- {run settings arguments to pass to vstest.console}
E.g.:  dotnet test foo.csproj --logger trx /Verbosity:detailed -- RunConfiguration.ResultsDirectory="C:\users\user\desktop\ResultsDirectory"